src/state/experiments/selectors.js

```diff
@@ -1,2 +1,29 @@
 export const getExperimentsCount = state =>
   state.get("experiments").get("items").size;
+
+export const getFilteredExperimentsByDate = (state) => {
+  let filteredExperiments = [];
+  const experiments = state.get("experiments").get("items"); 
+  const selectedStartTimestamp = getStartDatepickerTimestamp(state);
+  const selectedEndTimestamp = getEndDatepickerTimestamp(state);
+
+  for (let i = 0; i < experiments.size; i++) {
+    const experimentStartDate = experiments.getIn([i, "start_date"])
+    const experimentEndDate = experiments.getIn([i, "end_date"]);
+
+    if (experimentStartDate >= selectedStartTimestamp && selectedEndTimestamp >= experimentEndDate) {
+      filteredExperiments.push(experiments.getIn([i, "name"]))
+    } 
+  }
+  return filteredExperiments;
+}
+
+const getStartDatepickerTimestamp = (state) => {
+  const selectedStartDate = state.getIn(["dates", "startDate"]) || 0;
```

Comment:
So if the user hasn't chosen a start date, and it defaults to an empty string, which is falsy, then we `or` that with 0, so we get 0, then we create a date with that, which is `Jan 1 1970`, which is an arbitrary date the user didn't actually pick?  Isn't that a bit magical?  Why not just return `null`?